app/views/admin/characters/_relocate.haml

```diff
@@ -15,10 +15,10 @@
         %th.sub= f.label :user_id, 'New User Id'
         %td{class: cycle('even', 'odd')}= f.text_field :user_id, value: params[:user_id], placeholder: "New User", class: 'text'
       %tr
-        %td.sub With Templates
+        %th.sub With Templates
         %td.padding-5{class: cycle('even', 'odd')}
-          = f.check_box :include_templates, checked: params[:include_templates], class: 'vmid no-margin', style: 'margin-bottom: 3px;'
-          = f.label :include_templates, "Transfer Templates"
+          = f.check_box :include_templates, checked: params[:include_templates], class: 'vmid'
+          = f.label :include_templates, "Transfer Templates", class: 'vmid'
```

Comment:
tl;dr we're consistently using vmid (`vertical-align: middle`) on both the checkbox and the label, which makes the text height appropriate